tests/improver-weighted-blending/01-help.bats

```diff
@@ -52,6 +52,11 @@ Required for ChooseDefaultWeightsLinear: y0val and ONE of slope, ynval.
 Required for ChooseDefaultWeightsNonLinear: cval.
 
 positional arguments:
+  WEIGHTS_CALCULATION_METHOD
+                        Method to use to calculate weights used in blending.
+                        "linear": option to use the ChooseDefaultWeightsLinear
+                        plugin. "nonlinear": option to use the
+                        ChooseDefaultWeightsNonLinear plugin.
```

Comment:
I'm not sure I care which plugin is used, more what the method actually does. What is the non-linear method?
Is there precedent for just indicating the plugin used at this level?